frontend/src/components/worksheets/Worksheet/WorksheetHeader.js

```diff
@@ -67,6 +68,13 @@ export default ({
                                 onChange={() => reloadWorksheet()}
                                 allowASCII={true}
                             />
+                            {/*Hide invalid last modified date*/}
+                            {info && info['date_last_modified'] ? (
+                                <div style={{ fontWeight: 'normal', fontSize: 'small' }}>
+                                    Last Modified Date:{' '}
+                                    {convertDatetimeToLocalDatetime(info['date_last_modified'])}
```

Comment:
If we follow the suggestion I mentioned above, we could maybe even get rid of `convertDatetimeToLocalDatetime` altogether, and just replace by inlining it: `new Date(...).toString()`

Reply:
Change applied